spec/javascripts/services/short_form_data_service_spec.coffee

```diff
@@ -1,4 +1,4 @@
-do ->
+ do ->
```

Comment:
is this indentation correct? Looks like it's got one single space before `do` here